vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -471,6 +471,17 @@ public Connection getWatchLaterConnection() {
         }
         return null;
     }
+
+    @Nullable
+    public Connection getUsersWithAccessConnection() {
+        if (mMetadata != null
+            && mMetadata.mConnections != null
+            && mMetadata.mConnections.getUsersWithAccess() != null) {
+            return mMetadata.mConnections.getUsersWithAccess();
+        }
+        return null;
```

Comment:
This could be simplified with one less condition, as there is no need to null check `getUsersWithAccess` since the method is annotated `@Nullable`.
```java
if (mMetadata != null && mMetadata.mConnections != null) {
     return mMetadata.mConnections.getUsersWithAccess();
}
return null;
```